README.md

```diff
@@ -1,3 +1,8 @@
+<!-- Copyright 2020-2021 LunarG, Inc. -->
+<!-- Copyright 2020-2021 PlutoVR Inc. -->
+<!-- Copyright 2020 The Khronos Group Inc. -->
```

Comment:
Khronos can have a copyright here if we really want to give them one, I guess, but only LunarG and Pluto VR have contributed to README.md.